### PR TITLE
review chapter 11

### PR DESCRIPTION
**Pull request by Xronophobe**
Review 77% done. Currently stuck with a bug which I'll get back to.

Todo elaborate:
- no git commits in the chapter
- ~add `-K` to all ansible-playbook commands~
- ansible cfg file so the ansible-playbook commands can be shorter?

### CODE REVIEW

File: chapter_11_ansible.asciidoc [hunk 1]
```diff
@@ -874,12 +941,24 @@ on the server:
...
 element: [id="id_list_table"]; [...]
 ----
+// CSANAD: I have a different error:
+//         AssertionError: 'To-Do' not found in 'Bad Request(400)'
+//
+// >>> from django.conf import settings
+// >>>
+// >>>
+// >>> settings.ALLOWED_HOSTS
+// ['"192.168.122.23"']
```
Comment (Xronophobe) [inline chapter_11_ansible.asciidoc:959]: I tried a few things to remove the double quotes but somehow it's always there. I'll get back to this problem later. All ideas are welcome.

File: chapter_11_ansible.asciidoc [hunk 2]
```diff
@@ -218,6 +232,9 @@ At my registrar, the control screens looked a bit like <<registrar-control-scree
 .Domain setup
 image::images/gandi_add_dns_a_record.png["Registrar control screen for adding a DNS record"]
 
+// CSANAD: due to technical reasons, I cannot check on Digital Ocean whether this
+//         screenshot needs to be update. Please, someone else have a look!
```
Comment (Xronophobe) [inline chapter_11_ansible.asciidoc:236]: *updated

File: chapter_11_ansible.asciidoc [hunk 3]
```diff
@@ -696,43 +738,52 @@ And here's how we use it in the provisioning script:
...
 Infrastucture-as-code tools like Ansible aim to be "declarative",
 meaning that, as much as possible, you specify the desired state that you want,
 rather than specifying a series of steps to get there.
 
-This concept goes along with the idea of "idempotency",
+This concept goes along with the idea of "idempotence",
 which is wanting to get the same result when you run something for the first time,
 vs running it again on later occations.
+// CSANAD: I would rephrase it a little:
+//         "which is doing something multiple times brings the same results as
```
Comment (Xronophobe) [inline chapter_11_ansible.asciidoc:776]: I meant "which means"

File: chapter_11_ansible.asciidoc (hunk 2 above)
Comment (hjwp) [reply chapter_11_ansible.asciidoc:236]: i litterally just made that screenshot last month so it's ok.

File: chapter_11_ansible.asciidoc [hunk 4]
```diff
@@ -303,7 +322,7 @@ for.
 
     - name: Install docker  #<1>
       ansible.builtin.apt:  #<2>
-        name: docker  #<3>
+        name: docker.io  #<3>
```
Comment (hjwp) [inline chapter_11_ansible.asciidoc:325]: i did not have to do this - are you definitely on the same version of ubuntu?  22.04 jammy?

Comment (hjwp) [reply chapter_11_ansible.asciidoc:325]: for the record - my mistake, yes.
